packages/components/src/components/post-header/post-header.scss

```diff
@@ -14,6 +14,25 @@
 :host {
   z-index: 1020;
   position: relative;
+
+  // The following variables will affect the header to minimize it on scroll
+  --global-header-top: calc(
+    (var(--global-header-height) - var(--global-header-reduced-height)) * -1
+  );
+  --local-header-top: calc(
+    -1 * (var(--local-header-height) - var(--main-navigation-height)) + var(--global-header-reduced-height)
+  );
+  --logo-height: calc(var(--global-header-height) - var(--header-scroll-top));
+  --global-controls-top: calc(min((var(--header-scroll-top) / 4), 20px) * -1);
+
+  // If focus is within the header (except the logo and the post-mainnavigation elements), the header should have be shown fully
+  &:has(.global-sub > *:not(.logo):focus-within),
+  &:has(.local-sub:focus-within) {
+    --global-header-top: 0;
+    --local-header-top: var(--global-header-height);
+    --logo-height: var(--global-header-height);
+    --global-controls-top: 0;
+  }
```

Comment:
I think we moved these variables to the styles package now?
Can you check that and implement your update there instead of adding these variables back to the component styles?